diskUtils.py

```
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import yaml
+import json
+import os
+from pathlib import Path
+
+def load_config_file(config_file_path='config.yml'):
+    with open(config_file_path, 'r') as configFile:
+        return yaml.load(configFile)
+
+def load_account_data(accounts_file_path):
+    if Path(accounts_file_path).is_file():
+        with open(accounts_file_path) as f:
+            return json.load(f)
+    else:
+        return None
+
+def write_accounts(accounts_file_path, accounts):
+    # Write raw json to file
+    with open(accounts_file_path, 'w') as outfile:
+        json.dump(accounts, outfile)
+
+'''
+In case of failure, the last processed account is cached to disk,
+check it here for where to start the extraction process.
+'''
+def get_extraction_progress(progress_file_path):
+    with open(progress_file_path, 'r') as progress_file:
+        progress_str = progress_file.readline()
+
+        progress_int = 0
+        try:
+            progress_int = int(progress_str)
+        except:
+            pass
```

This is bad practice. You're essentially ignoring all exceptions. I'd recommend defining your own exceptions for situations you anticipate or use built in python exceptions. At the very least, use except Exception: to avoid catching system errors like keyboard interrupt.

Also, log it.

---

I'm responding to the whole fleet of these comments here:

You're 100% correct this is less than ideal practice but I don't think it's worth the effort to improve more. The file IO is purely temporary data and in the event of needing to restart a script. In fact looking back at this, we probably don't even want the temp data to be stored at all. The only gain is being able to pick up where the script died instead of reprocessing accounts.

What do you think about removing the file IO entirely, resolving the try/except discussion? Benefits are pretty small.